hiera.yaml

```diff
@@ -0,0 +1,8 @@
+---
+version: 5
+hierarchy:
+  - name: "Default values"
+    path: common.yaml
+defaults:
+  data_hash: yaml_data
+  datadir: data
```

Comment:
File is missing a newline. Might want to check your text editor.

Reply:
was using vscode, but forgot the \n, will sort this tonight.